vue/sbc-common-components/src/components/SbcAuthMenu.vue

```diff
@@ -27,7 +27,7 @@
 <script lang="ts">
 import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
 import { LDClient } from 'launchdarkly-js-client-sdk'
-import { IdpHint, LoginSource, Pages } from '../util/constants'
+import { Role, IdpHint, LoginSource, Pages } from '../util/constants'
```

Comment:
I see this is used on line 197 (which isn't changed in this PR). How did this work before???